include/fluent-bit/flb_signv4.h

```diff
@@ -40,6 +40,7 @@ flb_sds_t flb_signv4_do(struct flb_http_client *c, int normalize_uri,
                         time_t t_now,
                         char *region, char *service,
                         int s3_mode,
+                        struct mk_list *unsigned_headers,  /* flb_slist */
```

Comment:
looks like this broke the signv4 unit tests...

Reply:
That's odd